src/main/java/one/util/streamex/EntryStream.java

```diff
@@ -893,6 +947,12 @@ public boolean noneMatch(BiPredicate<? super K, ? super V> predicate) {
      * @param <VV> a type of values to select.
      * @param clazz a class to filter the values.
      * @return the new stream
+     * @see #nonNull()
+     * @see #filter(Predicate)
+     * @see #remove(Predicate)
+     * @see #selectKeys(Class)
+     * @see #selectValues(Class)

```

Comment:
Also here it looks like the method links to itself. And in some other similar places as well.

Reply:
I can correct it if you want, this was just a faster way to "link" each filtering method to one another with just a big no-brainer Copy/Paste